### DISCUSSION TITLE
找不到回家的路，看見失智症長輩的徬徨

### DISCUSSION THREAD

**Discussion by 110302550** [1101-GS4719-C/Python-Community-GS4719-C#1149] [General]
https://www.thenewslens.com/article/174800
我覺得失智症是一群症狀的組合，他會影響認知功能，而隨著年齡越大，失智的風險就越大。我認為在臺灣的話，65歲以上的長輩中有30萬人患有失智症，而失蹤老人中有一半可能患有失智症，是老人失蹤的主要原因。

**Comment by alicia0725** [comment]
身邊家人有兩位老人都罹患了失智症，幸運的是，發現得早，所以現在有讓他們去上一些互動式課程，我認爲是有明顯減緩老化速度的，其中我爺爺被檢查出失智到現在大概兩年多，現在他雖然叫不出我的名字，但我很確定他是認得我的。至於走丟的部分，大部分被檢查出的失智症患者其實都有配戴有名字電話的手環或項鍊。因此我建議可以定期帶老人家去醫院檢查，越早確診對老人的報障越大
